README.md

```diff
@@ -1939,8 +1976,8 @@ Other Style Guides
     }
     ```
 
-  <a name="comments--todo"></a><a name="17.5"></a>
-  - [17.5](#comments--todo) Use `// TODO:` to annotate solutions to problems.
+  <a name="comments--todo"></a><a name="17.6"></a>
```

Comment:
So what number should I give the section that I insert in between? Or should I just add it at the end of the comments section? Sorry, this is my first time contributing to this repo :)